README.md

```diff
@@ -1 +1,24 @@
-# kubebuilder-plugin
\ No newline at end of file
+# Enable kubebuilder-plugin for operator-sdk
+
+Update for kubebuilder-plugin path in go-mod pointing to the latest release.
+
```

Comment:
You need to describe how we enable this in the operator-sdk codebase. In order to use this project right now they would need to clone the operator-sdk repo, update the `go.mod` file to point to this repo. And they have to modify operator-sdk to enable it.  See the changes I had to make 
https://github.com/jmrodri/operator-sdk/commit/aa3d8c2de1e8dab6b101b7f0f7a98c8424fbd64c

Those changes will NOT be committed to operator-sdk any time soon which is why folks have to manually do this.

Reply:
Tell them that in order to drive this plugin while it is in development, we want to integrate with the `operator-sdk`.